BrainPortal/app/models/data_provider.rb

```diff
@@ -1055,14 +1055,34 @@ def alive_cached_valid?
   #     is_fast_syncing                      => true/false
   #     content_storage_shared_between_users => true/false
   def for_api
+    extra_info = {};
```

Comment:
Don't do that.

Overriding for_api() is only necessary when we need to provdie information that is not directly an attribute. E.g. in the original case, this was here because `is_browsable` is not an actual attribute of the model, so we had to invoke the method at merge it to the hash returned by the standard for_api().

For normal attributes, just add the attributes to `api_attr_visilble` in each class. This class method builds lists of attributes to return and also inherits automatically attributes defined in superclasses (so you don't have to repeat user_id for exmaple)